### PR TITLE
Fix EZP-24053: Preview locks content object, and cause version edit conflict

### PR DESCRIPTION
**Pull request by lolautruche**
> https://jira.ez.no/browse/EZP-24053
> https://jira.ez.no/browse/EZP-24052

LegacyBridge part. See also https://github.com/ezsystems/ezpublish-kernel/pull/1216 .
This PR ensures that legacy website toolbar is loaded with appropriate template when in preview.

### CODE REVIEW

**Comment by lolautruche** [comment]
Review ping @andrerom @bdunogier @dpobel @yannickroger @pspanja @glye

File: bundle/Controller/WebsiteToolbarController.php [hunk 1]
```diff
@@ -67,10 +76,11 @@ public function websiteToolbarAction( $locationId, Request $request )
             return $response;
         }
 
+        $content = $this->loadContentByLocationId($locationId);
```
Comment (andrerom) [inline bundle/Controller/WebsiteToolbarController.php]: doesn't seem to be reused below, temporary change?

**Comment by andrerom** [comment]
+1

File: bundle/Controller/WebsiteToolbarController.php (hunk 1 above)
Comment (lolautruche) [inline bundle/Controller/WebsiteToolbarController.php]: You're right. I changed my mind and forgot this.

**Comment by yannickroger** [comment]
+1

**Pull request merged**